server_admin/topics/user-federation/ldap.adoc

```diff
@@ -51,6 +57,13 @@ UNSYNCED::
   It is up to you to figure out how to synchronize back to LDAP. This allows {project_name} deployments to support
   updates of user metadata on a read-only LDAP server.  This option only applies when you are importing users from LDAP into the local {project_name} user database.
 
+NOTE: When LDAP provider is created, the set of initial <<_ldap_mappers,LDAP mappers>> is created. The mappers are configured on a "best-effort" basis
+      based on the chosen combination of the `Vendor`, `Edit Mode`, and `Import Users` switches. For example in case of UNSYNCED edit mode, the mappers are pre-configured
+      in a way that particular user attribute is preferably read from the DB instead of from the LDAP. However when you later change the edit mode,
```

Comment:
add "a" before "particular"
change DB to database

Reply:
Will change as you suggested.